OneCommunityGlobal/HighestGoodNetworkApp
# Sriram - Add Projects Page Inventory sort button

# Description
![image](https://github.com/user-attachments/assets/792b3692-caf3-4d5c-801a-3d2483893e1b)

# Main changes
- Updated the inventory sorting to use a new field `inventoryModifiedTime` to sort.


## Related PRS (if any):
This frontend PR is related to this [backend PR](https://github.com/OneCommunityGlobal/HGNRest/pull/1187).

# How to test
- check into current branch and this [backend branch](https://github.com/OneCommunityGlobal/HGNRest/pull/1187).
- do npm install and ... to run this PR locally
- Clear site data/cache
- log as admin user
- go to Other Links -> Projects
- verify that the table is being sorted properly when inventory sort button is clicked
